block.go

```diff
@@ -218,6 +218,7 @@ type Block interface {
 	GetHasChildren() bool
 	GetArchived() bool
 	GetParent() *Parent
+	GetRichText() string
```

Comment:
Let's rename `GetRichText` to `GetRichTextString`, because this function returns `string` instead of `notionapi.RichText`

Reply:
Its okay. Sure, I'm updating the PR and merging again.